src/Main.js

```diff
@@ -0,0 +1,177 @@
+        let minutes = String(today.getMinutes());
+        if (minutes.length === 1){
+            minutes = "0" + minutes;
+        }
+        p.innerHTML = today.getDate() + "/" + 
+                    (today.getMonth()+1) + "/" + 
+                    today.getFullYear() + "<br/>" + 
+                    today.getHours() + ":" + 
+                    minutes;
+        p.className = "time";
+        p.id = today;
+        p.style.fontSize = '10px';
+        return p;
+    };
+
+    // clear all tasks
+    clearAll(){
+        const taskSection = document.body.querySelectorAll("div.taskSection");
+        taskSection.forEach(elem => {
+            elem.classList.add("removed");
+            setTimeout(()=> {
+                elem.remove();
+                this.itemManager.removeTask(elem.id);
+            }, 800);
+        });
+    };
+
+    // sort tasks by date they created
+    sortTime(){
+        const taskSections = document.body.querySelectorAll("div.taskSection");
+        let ts_array = [].map.call(taskSections, function(elem){return elem;});
+        ts_array.sort((a,b) => {
+            a = a.querySelector("div.tasks").querySelector("p.time").id;
+            b = b.querySelector("div.tasks").querySelector("p.time").id;
+            if (this.asc_time){
+                return new Date(a) - new Date(b);
+            }
+            return new Date(b) - new Date(a);
+        });
+        this.asc_time = !this.asc_time;
+        for (let i=0; i < ts_array.length; i++){
+            taskSections[i].parentNode.appendChild(ts_array[i]);
+        }
+    };
+
+    // sort tasks by name
+    sortAZ(){
+        const taskSections = document.body.querySelectorAll("div.taskSection");
+        let ts_array = [].map.call(taskSections, function(elem){return elem;});
+        ts_array.sort((a,b) => {
+            a = a.querySelector("div.tasks").querySelector("p").innerText;
+            b = b.querySelector("div.tasks").querySelector("p").innerText;
+            if (this.asc_AZ){
+                return a.localeCompare(b);
+            }
+            return b.localeCompare(a);
+        });
+        this.asc_AZ = !this.asc_AZ;
+        for (let i=0; i < ts_array.length; i++){
+            taskSections[i].parentNode.appendChild(ts_array[i]);
+        }
+    };
+
+    // adding the event listeners for buttons functions
+    init(){
+        const addBtn = document.getElementById("addBtn");
+        addBtn.addEventListener("click", () => {
+            this.addTask(document.getElementById("newTask"));
+        });
+        const input = document.getElementById("newTask");
+        // can add tasks using the Enter key
+        input.addEventListener("keypress", (event) => {
+            if (event.key === "Enter"){
+                this.addTask(input);
+            }
+        });
+        const clearAllBtn = document.getElementById("clearAll");
+        clearAllBtn.addEventListener("click", () => {
+            this.clearAll();
+        });
```

Comment:
just a shorter version for those cases 😃 
```suggestion
        clearAllBtn.addEventListener("click", this.clearAll);
```